leakcanary-android-proguard/src/main/java/com/squareup/leakcanary/proguard/CopyProguardMappingFileTask.kt

```diff
@@ -0,0 +1,39 @@
+package com.squareup.leakcanary.proguard
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class CopyProguardMappingFileTask : DefaultTask() {
+
+  private val assetsOutputDirectory = "${project.buildDir}/generated/assets/"
+
+  @Input var variantDirName: String? = null
+
+  @Input @Optional var mappingFile: File? = null
+
+  @get:OutputDirectory
+  val leakCanaryAssetsOutputDirectory: File
+    get() = project.file("$assetsOutputDirectory$variantDirName")
+
+  init {
+    group = "leakcanaryplugin"
```

Comment:
why is this the group / where does this later show up?

Reply:
It's displayed for example in the output of $ gradle -t to output all the tasks of the same group together. I added this because I thought that maybe there will be more leak canary tasks in the future and they should be grouped together. But thinking about it again I guess we can remove it and add in the future if needed.

Reply:
I find it interesting to have a group, when listing tasks they are grouped. Maybe a more general group such as "memory monitoring" would provide more insights on what the tasks are doing.